src/veh_interact.h

```diff
@@ -77,6 +77,7 @@ class veh_interact
         int name_w;
 
         vehicle *veh;
+        bool has_screwdriver;
```

Comment:
Potentially beyond the scope of your PR but we could consider a more generic implementation specifying required tool qualities (eg. `SCREWING 2`, `BOLT TURNING 1`) rather than `has_foo()`